src/main/java/net/kodehawa/mantarobot/commands/currency/item/ItemHelper.java

```diff
@@ -469,7 +446,7 @@ public static boolean handleEffect(PlayerEquipment.EquipmentType type, PlayerEqu
     }
 
     public static Item getBrokenItemFrom(Item item) {
-        for (Item i : ItemReference.ALL) {
+        for (Item i : getBrokenItems()) {
```

Comment:
We don't need to check if the item is instance of Broken if we already get a broken items, I believe?